client/src/App.jsx

```diff
@@ -11,9 +11,10 @@ import HostMenu from './components/HostMenu.jsx';
 import RegisterUser from './components/RegisterUser.jsx';
 import LoggedIn from './components/LoggedIn.jsx';
 import Session from './components/Session.jsx';
+import PayBill from './components/PayBill.jsx';
 
 class App extends React.Component {
-  constructor (props) {
```

Comment:
I just reconfigured the rules for eslint, now it wont have error alarm for any format of pren before function 

        "space-before-function-paren": [
            "warning",
            "never"
        ]

Reply:
OK, I think we were all formatting it based on the original rule that asked for a space before function parentheses. Not a huge deal but this may mean we have slightly different syntax across files if we remove the rule now. But as long as it won't give us a bunch of red squiggly lines then we're good!